### PR TITLE
updated reference to IDD documentation

### PR DESCRIPTION
**Pull request by gundarev**

### CODE REVIEW

**Comment by gundarev** [comment]
#sign-off

File: windows/deployment/planning/windows-10-deprecated-features.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The features described below are no longer being actively developed, and might b
...
+| XDDM-based remote display driver  | Starting with this release, the Remote Desktop Services uses a Windows Display Driver Model (WDDM) based Indirect Display Driver (IDD) for a single session remote desktop. The support for Windows 2000 Display Driver Model (XDDM) based remote display drivers will be removed in a future release. Independent Software Vendors that use an XDDM-based remote display driver should plan a migration to the WDDM driver model. For more information on implementing remote display indirect display driver check out [Updates for IddCx versions 1.4 and later](/windows-hardware/drivers/display/iddcx1.4-updates). | 1903 |
```
Comment (illfated) [inline windows/deployment/planning/windows-10-deprecated-features.md:36]: Minor detail: readability comma:
```suggestion
| XDDM-based remote display driver  | Starting with this release, the Remote Desktop Services uses a Windows Display Driver Model (WDDM) based Indirect Display Driver (IDD) for a single session remote desktop. The support for Windows 2000 Display Driver Model (XDDM) based remote display drivers will be removed in a future release. Independent Software Vendors that use an XDDM-based remote display driver should plan a migration to the WDDM driver model. For more information on implementing remote display indirect display driver, check out [Updates for IddCx versions 1.4 and later](/windows-hardware/drivers/display/iddcx1.4-updates). | 1903 |
```
The only minor inconvenience is that the relative link (`/windows-hardware/drivers/display/iddcx1.4-updates`) does not work between the local pages here on https://github.com/MicrosoftDocs/windows-itpro-docs/ , but it works as intended on docs.microsoft.com .

**Review by illfated — APPROVED** [review]
Target page on docs.microsoft.com:
- https://docs.microsoft.com/en-us/windows/deployment/planning/windows-10-deprecated-features

Looks good to me.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - only @illfated's suggested change.

**Comment by gundarev** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by greg-lindsay — APPROVED** [review]

**Pull request merged**